TeslaApp/TeslaApp/StartScreen/StartView.swift

```diff
@@ -0,0 +1,156 @@
+// StartView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Start view
+struct StartView: View {
+    // MARK: - Constants
+
+    private enum Constants {
+        static let backgroundUnlockFirstColorName = "backgroundUnlock_first"
+        static let backgroundUnlockSecondThirdColorName = "backgroundUnlock_second_third"
+        static let backgroundUnlockFourthColorName = "backgroundUnlock_fourth"
+        static let backgroundLockFirstColorName = "viewTopGradient"
+        static let backgroundLockSeconColorName = "backgroundLock_second"
+        static let teslaLockedImageName = "teslaLocked"
+        static let gearshapeImageName = "gearshapeIcon"
+        static let lockText = "Lock"
+        static let unlockText = "Unlock"
+        static let unlockedImageName = "unlockedIcon"
+        static let lockedImageName = "lockedIcon"
+        static let teslaUnlockedImageName = "teslaUnlocked"
+        static let hiText = "Hi"
+        static let welcomeBackText = "Welcome back"
+    }
+
+    // MARK: - Public Properties
+
+    var body: some View {
+        NavigationView {
+            ZStack {
+                if startViewModel.isCarUnlocked {
+                    unlockedGradient
+                        .edgesIgnoringSafeArea(.all)
+                } else {
+                    lockedGradient
+                        .edgesIgnoringSafeArea(.all)
+                }
```

Comment:
```suggestion
        ZStack {
                startViewModel.isCarUnlocked ? unlockedGradient :  lockedGradient  
                } 
                 .edgesIgnoringSafeArea(.all)
```

можно попробовать так условие задать